ts/components/ui/utils/hooks.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+import Animated, {
+  Extrapolate,
+  interpolate,
+  interpolateColor,
+  useAnimatedStyle,
+  useDerivedValue,
+  useSharedValue,
+  withSpring
+} from "react-native-reanimated";
+import { IOColors, hexToRgba, useIOTheme } from "../../core/variables/IOColors";
+import {
+  IOScaleValues,
+  IOSpringValues
+} from "../../core/variables/IOAnimations";
+
+export const useDefaultListItemSpringAnimation = () => {
```

Comment:
Just a naming suggestion to be less verbose. We're not going to add different spring animations to the list items in the medium term.
```suggestion
export const useListItemSpringAnimation = () => {
```